Registration/RegisterWithAzure.psm1

```diff
@@ -977,10 +977,6 @@ Removes the activation resource created during New-AzsActivationResource
 Prompts the user to log in to the Azure Stack Administrator account, finds and removes the activation resource created
```

Comment:
Perhaps update this documentation

Reply:
User needs to provide AzureStack PrivilegedEndpoint credential so that Azure Stack is de-activated.

Reply:
Changing this to: Prompts the user to log in to the Azure Stack Administrator account, and deactivates the activation properties created by New-AzsActivationResource.